FordLabs/peoplemover
# Consolidate unassigned drawer tests

## Issue
Resolves N/A

## What was done
- [x] Moved all tests out of Unassigned.test.tsx that tested the UnassignedDrawer into the UnassignedDrawer.test.tsx file
- [x] Ensure all tests in Unassigned.test.tsx that rendered the whole app were tested in cypress or other unit tests and deleted the file

## How to test
1. Ensure that all tests that were in Unassigned.test.tsx that tested the UnassignedDrawer are now in UnassignedDrawer.test.tsx
2. Ensure that all things tested in Unassigned.test.tsx that required rendering the entire app are now only tested in cypress.
